.gitignore

```diff
@@ -0,0 +1,11 @@
+# Pycharm developers
+.idea/
+
+# Mac computers suck
```

Comment:
True, but probably shouldn't be included in the commit ;)

Reply:
Fixed!